bip-0053.mediawiki

```diff
@@ -6,7 +6,7 @@
   Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-0053
   Status: Draft
   Type: Standards Track
-  Created: ?
+  Created: 2024-12-20
```

Comment:
According to [BIP2](https://github.com/bitcoin/bips/blob/master/bip-0002.mediawiki#bip-header-preamble), the "Created" header records the date a proposal was assigned a number, which is why I suggested 2025-04-11.

![image](https://github.com/user-attachments/assets/5ac62de5-00b6-455e-940f-f22c88d228e1)